src/lib/style/bootstrap4_backports/_base-import.scss

```diff
@@ -15,6 +15,7 @@
   $input-border-color: SRC.$border-color-gray;
   $input-border-width: 1px;
   $input-border-radius: 2px;
+  $modal-footer-margin-between: 0rem;
```

Comment:
By default, this bootstrap 4 variable is nonzero, and adds margin around each item in the modal footer. The margin causes buttons to be improperly offset from the right edge of the footer content